npm/react/src/mount.ts

```diff
@@ -90,28 +79,20 @@ export const mount = (jsx: React.ReactNode, options: MountOptions = {}) => {
 
     reactDomToUse.render(reactComponent, el)
 
-    if (logInstance) {
-      const logConsoleProps = {
-        // @ts-ignore protect the use of jsx functional components use ReactNode
-        props: jsx.props,
-        description: 'Mounts React component',
-        home: 'https://github.com/cypress-io/cypress',
-      }
-      const componentElement = el.children[0]
-
-      if (componentElement) {
-        // @ts-ignore
-        logConsoleProps.yielded = reactDomToUse.findDOMNode(componentElement)
-      }
-
-      logInstance.set('consoleProps', () => logConsoleProps)
-
-      if (el.children.length) {
-        logInstance.set(
-          '$el',
-            (el.children.item(0) as unknown) as JQuery<HTMLElement>,
-        )
-      }
+    if (options.log !== false) {
```

Comment:
super nit: Compare against `true` unless `options.log` is more than just `boolean | undefined`

Reply:
`undefined` means `true` here :D

Reply:
Oh duh. Sorry